static/js/add-shows/new-show.js

```diff
@@ -150,7 +150,7 @@ MEDUSA.addShows.newShow = function() {
 
     $('#addShowButton').click(function() {
         // if they haven't picked a show don't let them submit
-        if (!$('input:radio[name="whichSeries"]:checked').val() && $('input:hidden[name="whichSeries"]').val().length !== 0) {
+        if (!$('input:radio[name="whichSeries"]:checked').val() && $('input:hidden[name="whichSeries"]').val().length == 0) {
```

Comment:
`$('input:hidden[name="whichSeries"]').val().length == 0`
`==` needs to be `===` for the tests to pass.
 
Can you change that, please?

Reply:
Done.